app/components/Reactions/Reaction.tsx

```diff
@@ -91,28 +91,36 @@ const Reaction = ({
     return <></>;
   }
 
-  let tooltipContent =
-    showPeople && optimisticUsers?.map((user) => user.fullName).join(', ');
-  tooltipContent = tooltipContent ? `${tooltipContent} reagerte med ` : '';
-  tooltipContent += emoji;
+  const tooltipContentNames =
+    showPeople &&
+    optimisticUsers
+      ?.map((user) => user.fullName)
+      .sort()
+      .map((name) => <div key={name}>{name}</div>);
+  const tooltipContent = tooltipContentNames ? (
+    <div>
+      {tooltipContentNames}
+      <p>har reagert med {emoji}</p>
+    </div>
+  ) : (
+    <div></div>
```

Comment:
Are we sure we want an empty tooltip? Could we not just pass `undefined`? Is there any logic in the Tooltip component for this?

Reply:
Fixed, returns undefined, I am not sure about the logic part.